public/js/test/mochitest/head.js

```diff
@@ -95,6 +95,33 @@ function waitForState(dbg, predicate) {
   });
 }
 
+function waitForMs(time) {
+  return new Promise(resolve => setTimeout(resolve, time));
```

Comment:
We already have this as a shared utility: `waitForTime`. Note that we should never actually use it in tests except for debugging :)